### PR TITLE
Contribute workbook for Azure Logic Apps

### PR DESCRIPTION
**Pull request by karanbirsingh**
This is a workbook for Azure Logic Apps standard that will be referenced from the LA portal in the insights blade.

It was tested by deploying to a custom gallery, and then via `enableWorkbooks` + `apply` on the resource. There are three tabs, describing overview, workflows, and compute.

Please help advise:
- For CODEOWNERS, will an @azure team work? Or do we need to provide a @microsoft one?
- Do we need a gallery file update for our scenario? Or is contributing the workbook sufficient if we reference it directly from the built-in blade?

Feel free to ping me at `karansin`

https://github.com/microsoft/Application-Insights-Workbooks/assets/7775527/d476e644-77d5-4ff9-b9c8-12289b0403b0

## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
* [x] Validate your changes using one or more of the [testing](../Documentation/Testing.md) methods.

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [ ] ensure your template has a corresponding gallery entry in the gallery folder
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

**Comment by gardnerjr** [comment]
for CODEOWNERS, i believe has to be in the microsoft org (azure is its own org?)  you could try it, and then hopefully github would say "this codeowners file is invalid" in your branch and tell you that it can't find those users?


you don't show it in the video (the blade is all cropped out) so i'm not sure if you're showing this in WorkbookViewerBlade or something else?  if the blade has a "Workbooks" button that either shows or navigates to a gallery somewhere, your template should be visible there?

**Comment by matteing** [comment]
Hi Karanbir! Reached out through Teams with a _slightly tangential question_ about how this workbook was tested. 😄

**Comment by karanbirsingh** [comment]
Thanks @gardnerjr 

> for CODEOWNERS, i believe has to be in the microsoft org (azure is its own org?) you could try it, and then hopefully github would say "this codeowners file is invalid" in your branch and tell you that it can't find those users?

Got it - I'll create/find a @microsoft team - testing results were unclear but [this GH issue suggests it's unsupported](https://github.com/orgs/community/discussions/21781)

> you don't show it in the video (the blade is all cropped out) so i'm not sure if you're showing this in WorkbookViewerBlade or something else? if the blade has a "Workbooks" button that either shows or navigates to a gallery somewhere, your template should be visible there?

Here's a better screenshot:
image

This is with a feature flag on portal-side that uses the `WorkbookViewerBlade` to point to the private test gallery in the insights item. Happy to share the code - I actually don't see a workbooks button here.

File: CODEOWNERS [hunk 1]
```diff
@@ -127,6 +127,8 @@
 /Workbooks/Azure*Private*MEC/** @microsoft/azure-private-mec-workbooks
 /Workbooks/Azure*API*Management/** @microsoft/azure-api-management
 
+/Workbooks/Azure*Logic*Apps/** @Azure/azure-logicapps-team
```
Comment (gardnerjr) [inline CODEOWNERS:130]: there you go, codeowners reports this as invalid:

> Unknown owner on line 130: make sure the team @Azure/azure-logicapps-team exists, is publicly visible, and has ?> write access to the repository
>  …Workbooks/Azure*Logic*Apps/** @Azure/azure-logicapps-team

so if you don't already have a team in @microsoft org you need to add one, and then let me know what it is so i can add that team to have write access to the repo.

technically this shoudl have been one of the first steps as it is in the contributing docs:
https://github.com/microsoft/Application-Insights-Workbooks/blob/master/CONTRIBUTING.md#recommended-setup-for-contributing

> If you are a Microsoft employee:

>Ensure your github user is already a member of the [Microsoft organization](https://repos.opensource.microsoft.com/orgs)
> Look at the CODEOWNERS file to find if there is already a team set up for the area you are looking to contribute.
> If there is an existing team, ask the owners of that team to add you there if appropriate.
> if there is not an existing team, and you're looking to add workbook templates to new areas, in most cases you'll need to create a new team in the Microsoft organization in GitHub.
> If you created a new team, or you don't fall under one of the existing teams, send mail to azmonworkbooks with all the relavant information, including GitHub team name and details about the area you want to contribute to, so we can direct your request to the appropriate teams.

Comment (karanbirsingh) [reply CODEOWNERS:130]: Thanks! I did read those docs, but thought perhaps [these other MSFT orgs](https://github.com/enterprises/microsoftopensource/organizations) might be part of the 'in most cases' 🙂 Will send the new team reference soon

**Review by gardnerjr — APPROVED** [review]

**Pull request merged**